lib/OpenQA/WebAPI.pm

```diff
@@ -74,6 +74,29 @@ sub startup {
     # Set some application defaults
     $self->defaults(appname => $self->app->config->{global}->{appname});
 
+    # Get application version
+    if (-e $self->home->child('.git') and -e $self->home->child('.git/refs/heads/master')) {
```

Comment:
git is called on initial start of the application - there is no user input whatsoever.